command/harness/setup.go

```diff
@@ -222,5 +230,11 @@ func HandleSetup(ctx context.Context, r *SetupVMRequest, s store.StageOwnerStore
 
 	logr.WithField("response", fmt.Sprintf("%+v", setupResponse)).Traceln("VM setup is complete")
 
+	deleteInfoMap["shouldDelete"] = "true"
```

Comment:
we should use a bool instead of string for true or false.

Reply:
create env state takes a string to string mapping map